src/CMakeLists.txt

```diff
@@ -100,6 +100,40 @@ if(${TRITON_ENABLE_S3})
   message(STATUS "Using aws-sdk-cpp ${AWSSDK_VERSION}")
 endif()
 
+#
+# Azure Storage
+#
+if(${TRITON_ENABLE_AZURE_STORAGE})
+include(FetchContent)
+FetchContent_Declare(
+    azuresdk
+    GIT_REPOSITORY      https://github.com/Azure/azure-sdk-for-cpp.git
+    GIT_TAG             azure-storage-blobs_12.7.0
+)
+FetchContent_GetProperties(azuresdk)
+if(NOT azuresdk_POPULATED)
+    # [FXIME] WAR for configuring compiler option
+    if (DEFINED CMAKE_POSITION_INDEPENDENT_CODE)
+      set(OLD_CMAKE_POSITION_INDEPENDENT_CODE ${CMAKE_POSITION_INDEPENDENT_CODE})
+    endif()
+    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+    FetchContent_Populate(azuresdk)
+    # Adding all Azure SDK libraries
+    set(DISABLE_AZURE_CORE_OPENTELEMETRY ON)
+    add_subdirectory(${azuresdk_SOURCE_DIR} ${azuresdk_BINARY_DIR} EXCLUDE_FROM_ALL)
+    # Adding one Azure SDK Library only (Storage blobs)
+    # add_subdirectory(${azuresdk_SOURCE_DIR}/sdk/core/azure-core ${azuresdk_BINARY_DIR}/sdk/core/azure-core EXCLUDE_FROM_ALL)
```

Comment:
Same comments as Jacky's above. Can we remove all this commented out code if it's not necessary long-term?